docs/plugins/accessingManagingState.md

```diff
@@ -0,0 +1,72 @@
+You can access and modify the following attributes of the carousel. To manage and access react-carousel, you need to install [**recoil**](https://recoiljs.org/docs/introduction/getting-started) library.
+
+```js
+import {
+  trackWidthState,
+  trackStylesState,
+  carouselValueState,
+  slideMovementState,
+  transitionEnabledState,
+  slideOffsetState,
+  slidesState,
+  slideWidthState,
+  activeSlideIndexState,
+  transformOffsetSelector,
+  nearestSlideSelector,
+  getCurrentValueSelector,
+} from '@brainhubeu/react-carousel';
+```
+
+example usage
+```js
+import { useRecoilState } from 'recoil';
+
+const customPlugin = () => ({
+  plugin: () => {
+    const [trackStyles, setTrackStyles] = useRecoilState(trackStylesState);
+
+      useEffect(() => {
+        setTrackStyles({
+          ...trackStyles,
+          marginLeft: myCustomFuncToModifyMargin(),
+        });
+      }, [carouselProps.value]);
+  },
+})
+```
+
+* ```trackWidthState: Number```: Width of the track container.
+
+* ```trackStylesState: Object```: Styles applied to the track container.
+
+* ```carouselValueState: Number```: Active carousel slide index.
+
+* ```slideMovementState: Object```: Used to calculate active slide.
+
+```js
+  key: '@brainhubeu/react-carousel/slideMovementState',
+  default: {
+    clicked: null, // index of the clicked slide
+    dragStart: null, // X position of drag event start
+    dragEnd: null, // X position of drag event end
+    dragOffset: 0, // Distance of the drag
```

Comment:
I wrote about a unification of uppercase/ lowercase in comments but I see my comment disappeared